### DISCUSSION TITLE
Some of my own architecture photos

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [BuddiesOfBudgie/budgie-backgrounds#36] [Background Suggestions]
![IMG_20230105_091139220_HDR](https://user-images.githubusercontent.com/105567407/211118042-029c26b2-1f35-4e07-932d-35e216a18337.jpg)
![IMG_20230102_163139087](https://user-images.githubusercontent.com/105567407/211118046-e057d465-efd0-4978-ac18-41ca470e75a1.jpg)
![IMG_20221230_094019909_HDR](https://user-images.githubusercontent.com/105567407/211118051-baea2f70-fad9-4fe1-8a72-183588c1bc9c.jpg)
-------
![88x31](https://user-images.githubusercontent.com/105567407/211118529-02d21931-f154-4e97-85bb-9f29c1f86913.png)
To the extent possible under law, Aaron C. has waived all copyright and related or neighboring rights to this work. This work is published from: United States.

**Comment by Drsheppard01** [comment]
it looks incredibly cool
